libraries/standard/json/src/json.c

```diff
@@ -160,9 +160,17 @@ static bool_ skipUTF8MultiByte( const char * buf,
                                 size_t max )
 {
     bool_ ret = false;
-    size_t i = *start, bitCount, j;
+    size_t i, bitCount, j;
     uint32_t value = 0;
-    uint8_t c = ( uint8_t ) buf[ i ];
+    uint8_t c;
+
+    assert( ( buf != NULL ) && ( start != NULL ) && ( max > 0U ) );
```

Comment:
`assert __CPROVER_r_ok` on the actual required size of the buffer

Reply:
I'd prefer to keep the proof separate.